src/server/nuclearnet/tests/fake_nuclearnet_client.tests.ts

```diff
@@ -0,0 +1,192 @@
+import { FakeNUClearNetClient } from '../fake_nuclearnet_client'
+import { FakeNUClearNetServer } from '../fake_nuclearnet_server'
+
+describe('FakeNUClearNetClient', () => {
```

Comment:
I don't use address/port anywhere in the fake, so hopefully I can ignore those things? I've updated the dummy values to be very fake such that nobody would be able to think they work.